File: src/main/java/org/example/chessplatformbe/domain/Comment.java
package org.example.chessplatformbe.domain;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Comment {

    private Integer id;

    private String text;

    private Integer userId;

    private Integer articleId;
}

File: src/domain/comment.rs
#[derive(Debug, Clone, Default)]
pub struct Comment {
    pub id: u32,
    pub text: String,
    pub user_id: u32,
    pub article_id: u32,
}

impl Comment {
    pub fn new(id: u32, text: String, user_id: u32, article_id: u32) -> Self {
        Self {
            id,
            text,
            user_id,
            article_id,
        }
    }
}
